src/main/java/net/mcjukebox/plugin/bukkit/listeners/RegionListener.java

```diff
@@ -83,10 +83,7 @@ public void onMove(PlayerMoveEvent e){
                 if (lastShow == null) {
                     //Region no longer exists, stop the music.
                     JukeboxAPI.stopMusic(e.getPlayer());
```

Comment:
With the return removed we're stopping the music twice. Maybe remove the stopMusic call on line 91, and the return on line 88 if we are going to use this approach?